maven-plugin/src/main/scala/akka/grpc/maven/GenerateMojo.scala

```diff
@@ -28,7 +28,9 @@ class GenerateMojo @Inject() (project: MavenProject) extends AbstractMojo {
   var generateServer: Boolean = _
 
   override def execute(): Unit = {
-    val schemas = findProtoFiles(new File(protoPath))
+    val protoDirectory = new File(project.getBasedir, protoPath).getAbsoluteFile
+    if (!protoDirectory.exists()) throw new RuntimeException(s"Configured protoPath $protoDirectory does not exist")
```

Comment:
I'd put `[ ]` around `$protoDirectory` to make it more clear if it's empty string (and consistent with how we do it many other places)

Reply:
:+1: - good idea to add the check!